### DISCUSSION TITLE
there we can add role and permission feature.

### DISCUSSION THREAD

**Discussion by krishantalwar** [prolinkinfo/InsuranceProCRM#2] [Ideas]
First of all Congratulations on your wonderful work.

Assume users as agents.
There we can add a feature that gives the user role and permission.

1) we need to permit some specific modules.
2) we need to permit the to agent add, edit, delete, and update levels on each module.
3) we need to make the head of some agents head of a specific policy type. so he can manage only those policies add or edit, delete,

there are more cases I just defined some. If you are interested we will start discussions

**Comment by prolinkinfo** [comment]
Thanks for your suggestions @krishantalwar 

Please feel free to add your ideas to this forum and we can add it to the CRM.
